<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="Free">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Used">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Total">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="StartButton">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>20</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BalanceButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>MyBalance</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="monitor1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>141</width>
     <height>241</height>
    </rect>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
   <property name="openLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Time">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>20</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="TradingRecord">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>1131</width>
     <height>221</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>COIN</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>LEVERAGE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ISOLATED</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ENTRYPRICE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SIZE(USDT)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>MARGIN(USDT)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PNL(USDT)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ROD%</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="CoinList">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>120</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>9</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>COIN</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="CoinAdd">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CoinName">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>80</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteButton">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>350</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CoinDelete">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>350</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>15분봉 근접 코인</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="monitor2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>171</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="monitor3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>181</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="monitor4">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>120</y>
     <width>171</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="monitor5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>181</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>RCI값</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>급등 코인</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
